.circleci/config.yml

```diff
@@ -5,15 +5,6 @@ workflows:
   build:
     jobs:
       - build
-      - lms_unit_tests:
-          requires:
-            -  build
-      - cms_unit_tests:
-            requires:
-            -  build
-      - lib_unit_tests:
-            requires:
-            -  build
```

Comment:
@symbolist why were this removed?

Reply:
@rocioar The current Circle CI plan has only 25 build hours per month so temporarily disabled them while getting js tests working. Will remove this commit before merging.